pkg/pipes/pipes.go

```diff
@@ -0,0 +1,182 @@
+package pipes
+
+import (
+	"flappy/pkg/bird"
+	"flappy/pkg/window"
+	"fmt"
+	"github.com/veandco/go-sdl2/img"
+	"github.com/veandco/go-sdl2/sdl"
+	"math/rand"
+	"sync"
+	"time"
+)
+
+const (
+	pipeTexturePath = "../resources/Pipe-3.png"
+	initialPipeX    = 800
+	pipeWidth       = 43
+	minPipeHeight   = 220
+	maxPipeHeight   = 340 // minPipeHeight + 120
+	pipeSpeed       = 10
+)
+
+type Pipes struct {
```

Comment:
💡  Is a good practice to declare in packages the exported types as interfaces and the concrete implementations as private structs (lowercase). This way, is easier for testing using gomock.
So for the `Pipes` you could have:
```golang
type Pipes interface{
   Destroy()
   Restart()
   Touch()
}

type pipes struct{
   //etc.
}

func (ps *pipes) Destroy() {
  //blablabla
}

// And the rest of methods
```